### PR TITLE
update size per tx

### PR DESCRIPTION
**Pull request by kroos47**
updated the tx size from 16KB to 512KB.

### CODE REVIEW

File: pallets/dactr/src/weights.rs [hunk 1]
```diff
@@ -18,20 +18,19 @@
 //! Autogenerated weights for da_control
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2023-06-28, STEPS: `20`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
-//! HOSTNAME: `ip-172-31-12-189`, CPU: `Intel(R) Xeon(R) Platinum 8175M CPU @ 2.50GHz`
-//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+//! DATE: 2023-06-30, STEPS: `50`, REPEAT: 25, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `ip-172-31-39-104`, CPU: `Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz`
+//! EXECUTION: Some(Native), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
```
Comment (fmiguelgarcia) [inline pallets/dactr/src/weights.rs:23]: Execution should be using `Wasm` 
BTW the reference machine's CPU has changed, which one should we use?

Comment (Leouarz) [reply pallets/dactr/src/weights.rs:23]: This machine is way more powerful than the one I was using, is it wanted ?

**Comment by Leouarz** [comment]
There are other places where the size need to be changed (also one test if i'm not mistaken)

**Review by Leouarz — APPROVED** [review]

**Comment by Leouarz** [comment]
Should we incvrease spec version ?

**Comment by prabal-banerjee** [comment]
> Should we increase spec version ?

Yes, unless there is some major concern. Also, let's make sure LCs are working with the updated spec version.

**Review by 0xSasaPrsic — APPROVED** [review]

**Pull request merged**